mobile/Screen/SubmissionScreen.js

```diff
@@ -199,26 +278,40 @@ function SubmissionScreen() {
 
     try {
       const res = await axios.post(`${URL}/submissions/post`, {
-        name,
-        email,
+        name: 'Ryan',
+        email: 'ryanswkim2003@gmail.com',
```

Comment:
when you integrate it with accounts, remember to use redux to get the user id and then call api route for get name and email (you can create a new route which would be pretty simple or let me know and I can do it for you)